public/api/conf/1.0/application.raml

```diff
@@ -111,61 +111,42 @@ traits:
-         to:
-           description: To date to search for VAT returns
-           type: string
-           example: 2017-01-25
-        responses:
-          200:
-            headers:
-              X-CorrelationId:
-                example: 5968c22d620000620096db5f
-                description: Unique id for operation tracking
-            body:
-               application/json:
-                 type: !include schemas/VatReturns.json
-                 example: !include examples/VatReturns.json
-          400:
-            body:
-              application/json:
-                type: types.errorResponse
-                examples:
-                  invalidNino:
-                    description: Invalid VRN
-                    value:
-                      code: VRN_INVALID
-                  invalidDateFrom:
-                    description: Invalid date from
-                    value:
-                      code: INVALID_DATE_FROM
-                  invalidDateTo:
-                    description: Invalid date to
-                    value:
-                      code: INVALID_DATE_TO
-                  invalidDateRange:
-                    description: Invalid date range
-                    value:
-                      code: INVALID_DATE_RANGE
-          403:
-            body:
-              application/json:
-                type: types.errorResponse
-                examples:
-                  dateRangeTooLarge:
-                    description: The start date cannot be more than four years from the current date.
-                    value:
-                      code: DATE_RANGE_TOO_LARGE
+      /{periodKey}:
+        uriParameters:
+          periodKey:
+            description: Period key.
+            type: string
+            example: "#001"
+        get:
+          displayName: View VAT Returns
+          description: Retrieve submitted VAT returns
+          is: [headers.acceptHeader, contentType, testScenarioHeader]
+          (annotations.scope): "write:vat"
+          securedBy: [ sec.oauth_2_0: { scopes: [ "write:vat" ] } ]
+          responses:
+            200:
+              headers:
+                X-CorrelationId:
+                  example: 5968c22d620000620096db5f
+                  description: Unique id for operation tracking
+              body:
+                 application/json:
+                   type: !include schemas/VatReturn.json
+                   example: !include examples/GetVatReturn.json
+            400:
+              body:
+                application/json:
+                  type: types.errorResponse
+                  examples:
+                    invalidNino:
+                      description: The provided VRN is invalid
+                      value:
+                        code: VRN_INVALID
+                    periodKeyInvalid:
+                      description: Invalid period key
+                      value:
+                        code: INVALID_PERIOD_KEY
```

Comment:
According to the confluence page -> https://confluence.tools.tax.service.gov.uk/pages/viewpage.action?spaceKey=MTE&title=View+VAT+Returns

The error code should be `PERIOD_KEY_INVALID`. Also another code returned by DES `VAT_RETURN_MISSING` is not being handled in the code.